lsp-clients.el

```diff
@@ -173,7 +173,8 @@ This directory shoud contain a file matching groovy-language-server-*.jar"
 (defun lsp-typescript-javascript-tsx-jsx-activate-p (filename &optional _)
   "Check if the javascript-typescript language server should be enabled based on FILENAME."
   (or (string-match-p (rx (one-or-more anything) "." (or "ts" "js") (opt "x") string-end) filename)
-      (derived-mode-p 'js-mode 'js2-mode 'typescript-mode)))
+      (and (derived-mode-p 'js-mode 'js2-mode 'typescript-mode)
+           (not (eq 'json-mode major-mode)))))
```

Comment:
This would be better to have it as `(not (derived-mode-p 'json-mode))` since there's `jsonc-mode` that's extended `json-mode` which support comment inside json

Reply:
Thank you, will be addressed before merging.